core/components/atoms/message/__stories__/variants/AppearanceTitle.story.jsx

```diff
@@ -4,19 +4,14 @@ import { Message, Text } from '@/index';
 // CSF format story
 export const appearanceWithTitle = () => {
   const appearances = ['alert', 'info', 'success', 'warning'];
-  const innerStyle = {
-    display: 'flex',
-    'align-items': 'center',
-    'flex-direction': 'column',
-  };
 
   const title = 'Outreach was not saved';
 
   return (
     <div className="Row">
       {appearances.map((appear, ind) => {
         return (
-          <div key={ind} style={innerStyle} className="mr-7 mb-7 w-25 d-flex">
+          <div key={ind} className="mr-7 mb-7 w-25 d-flex flex-column align-items-center">
```

Comment:
Please refactor the example to use `Row` and `Column` like https://codesandbox.io/s/trusting-https-ny9i1d?file=/index.js